### ISSUE TITLE
Latest versions do no catch errors

### ISSUE THREAD

**Issue by mskrip** [astral-sh/ruff#8093]
I tested this only on the line-length violation, but on multiple `ruff` versions (see below)

## Minimal code snippet:

`sample.file.py`:
```python
"""
This is a sample file with line that's just a bit too long and ruff should flag this as an error. Hopefully it does.
"""
```

Running `$ ruff check --isolated .` does not result in an error since version 0.1.0. This flags an error on 0.0.292.

```sh
$ ruff --version                                                                                                                                                                                                                         
ruff 0.0.292
$ ruff check --isolated .                                                                                                                                                                                                                
sample_file.py:2:89: E501 Line too long (116 > 88 characters)
Found 1 error.
$ echo $?
1
...
$ ruff --version
ruff 0.1.0
$ ruff check --isolated .
$ echo $?
0
...
$ ruff --version
ruff 0.1.1
$ ruff check --isolated .
$ echo $?
0
```

**Comment by charliermarsh** [comment]
Thanks for the clear issue. `line-too-long`  (`E501`) was removed from the default rule set in [v0.1.0](https://github.com/astral-sh/ruff/blob/main/CHANGELOG.md#breaking-changes), so I believe this is expected -- but you can re-add it with `--extend-select E` or by setting `extend-select = ["E"]` in your `pyproject.toml` or `ruff.toml` file.

**Comment by mskrip** [comment]
Oh, I completely missed that line in the changelog :facepalm: , thank you very much for the fast response. That solves it then.

**Comment by charliermarsh** [comment]
No worries! Sorry for the churn.
